examples/siro_sandbox/sandbox_app.py

```diff
@@ -261,8 +265,14 @@ def _env_step(self, action):
             self._metrics,
         ) = self.gym_habitat_env.step(action)
 
+    def get_num_episodes_done(self):
```

Comment:
I don't think these need to be public. And perhaps we don't need these getters at all? I.e. for constructing SandboxService, just do `lambda: self._num_episodes_done`?